back/engines/commercial/multi_tenancy/lib/tasks/core/clean_up_orphaned_image_records.rake

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+namespace :cl2back do
+  desc 'Remove image records not associated with resource or with nil value for image field'
+  # Usage:
+  # Dry run (no changes): cl2back:clean_up_orphaned_image_records
+  # Execute (destroys records!): cl2back:clean_up_orphaned_image_records['execute']
+  task :clean_up_orphaned_image_records, [:execute] => [:environment] do |_t, args|
+    live_run = true if args[:execute] == 'execute'
+
+    Tenant.switch_each do |tenant|
+      puts "Processing images for tenant #{tenant.name}"
+
+      # ContentBuilder layout_images
+      # Find all image codes used in all layouts for all projects, then destroy any layout_images with code not in use,
+      # and that are more than 3 days old.
+      image_codes = []
+
+      ContentBuilder::Layout.all.each do |layout|
+        image_codes += ContentBuilder::LayoutService.new.images(layout).pluck(:code)
+      end
+
+      ContentBuilder::LayoutImage.where.not(code: image_codes).each do |image|
+        # layout_images are created whenever an admin adds an image to a layout form, regardless of whether that image
+        # is eventually referenced by a layout (when / if the layout is saved).
+        # By only destroying unused layout_images with an age of 3+ days, we can be reasonably confident that the
+        # admin does not intend to add the image to a layout, and the image is truly orphaned.
+        if image.created_at < 3.days.ago
```

Comment:
@sebastienhoorens Could you please let us know if we should change this interval or if we should not delete these layout images at all?

Reply:
@alexander-cit I think this makes sense. It can happen that images were added, but the content layout was never saved. I think it's fine to require the admin to save their layouts within 3 days.

Reply:
FYI: It also happens when an image is replaced in a layout.